js/android/app/src/main/AndroidManifest.xml

```diff
@@ -46,12 +47,6 @@
             <data android:scheme="berty" />
         </intent-filter>
       </activity>
-      <meta-data
-        android:name="com.shakebugs.APIClientID"
-        android:value="${SHAKE_API_ID}" />
-      <meta-data
-        android:name="com.shakebugs.APIClientSecret"
-        android:value="${SHAKE_API_SECRET}" />
```

Comment:
Are you sure about this?

Reply:
No I need to add it back, I left android for now since we focus on iOS and we need to merge this asap